src/syrupy/report.py

```diff
@@ -48,9 +50,23 @@ class SnapshotReport(object):
     updated: "SnapshotFossils" = attr.ib(factory=SnapshotFossils)
     used: "SnapshotFossils" = attr.ib(factory=SnapshotFossils)
 
+    def filter_fossils(self, item: "SnapshotFossil") -> bool:
+        if not self.ran_test_files or not self.is_providing_paths:
```

Comment:
Shouldn't this just be `self.ran_item` since `ran_test_files` is a one to one mapping of the test locations of the ran items